src/nvim/menu.c

```diff
@@ -1177,7 +1178,7 @@ static char_u *menu_text(const char_u *str, int *mnemonic, char_u **actext)
   if (p != NULL) {
     if (actext != NULL)
       *actext = vim_strsave(p + 1);
-    text = vim_strnsave(str, (int)(p - str));
+    text = vim_strnsave(str, (size_t)(p - str));
```

Comment:
Is `p` guaranteed to be greater than `str`? May be easiest to assert above this.